docs/howto/how-to-cross-build-runtime-for-aarch64.md

```diff
@@ -1,2 +1,102 @@
-# How to Cross-build Runtime for Aarch64
+# How to Cross-build Runtime for AARCH64
 
+In **ONE**, we use `AARCH64` on build files such as Makefile, CMakeLists.txt and so on.
+
+## Prepare AARCH64 Ubuntu RootFS
+
+Install required packages
+
+```
+$ sudo apt-get install qemu qemu-user-static binfmt-support debootstrap
+```
+
+Use `build_rootfs.sh` script to prepare Root File System. You should have `sudo`
+
+```
+$ sudo ./tools/cross/build_rootfs.sh aarch64
+```
+- supports `arm`(default) and `aarch64` architecutre for now
+- supports `xenial`(default) and `trusty` release
```

Comment:
supports `xenial`(default), `trusty` and `bionic` release

Reply:
@hseok-oh , updated. Thanks!